pkg/chunkenc/memchunk_test.go

```diff
@@ -251,11 +252,12 @@ func TestRoundtripV2(t *testing.T) {
 	for _, f := range HeadBlockFmts {
 		for _, enc := range testEncoding {
```

Comment:
hmm why didn't you have to assign `enc := enc` here?

Reply:
Yeah, it's probably also needed. Govet might not catch every instance. https://github.com/golang/go/issues/16520 is the issue that introduced this. It might just work for 1 loop variable, not multiple nested loops.